uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_manage-leave-requests.scss

```diff
@@ -57,20 +84,52 @@
  * @TODO remove this patch when Shoreditch theme is migrated to SSP
  */
 .chr_manage_leave_requests {
-  .btn.ui-select-toggle {
-    padding: 4px 10px !important;
-    text-transform: none !important;
+  &[data-role="admin"] {
+    .ui-select-toggle {
```

Comment:
Actually, not only this. In SSP it is off:

![image](https://user-images.githubusercontent.com/3973243/28062188-067ab2f8-6625-11e7-81f1-e10d359f1db1.png)

Can I apply a additional rule to SSP only?

```css
.ui-select-toggle {
  padding: 7px 12px !important;
  font-size: 100% !important;
}
```